pkg/builder/builder.go

```diff
@@ -264,7 +264,9 @@ func loadValues(ctx *Context) error {
 }
 
 func archiveSrc(ctx *Context) error {
-	contextDir, relDockerfile, err := build.GetContextFromLocalDir(ctx.AppDir, ctx.Env.Dockerfile)
+
+	dockerfilePath := filepath.Join(ctx.AppDir, ctx.Env.Dockerfile)
+	contextDir, relDockerfile, err := build.GetContextFromLocalDir(ctx.AppDir, dockerfilePath)
```

Comment:
Before this change, the second parameter passed to `build.GetContextFromLocalDir` could have been the empty string, and the default Dockerfile would have been `Dockerfile`.

Now, we surely pass at least the application directory, so it will never be the empty string - so Docker expects that path to be an actual Dockerfile, resulting in #808